src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.ts

```diff
@@ -91,19 +91,24 @@ export class RbdListComponent implements OnInit {
     const editAction: CdTableAction = {
       permission: 'update',
       icon: 'fa-pencil',
+      disable: (selection: CdTableSelection) =>
```

Comment:
This is not needed. Table Actions already takes care of disabling "update" and "delete" actions.